### ISSUE TITLE
Is this project dead? :(

### ISSUE THREAD

**Issue by michalczaplinski** [sdiehl/write-you-a-haskell#93]

**Comment by lohmander** [comment]
Agreed! If time and money is sparse, I'd not be opposed to starting some sort of crowd funding. Really would love to have this book completed!

**Comment by tbenst** [comment]
+1 for crowd funding, would chip in if someone organizes. Quite a unique book as far as I can tell, no alternatives!

**Comment by AgustinCB** [comment]
+1 I'd also love to donate money for the project if it keeps it moving.

**Comment by caasi** [comment]
+1 I would back this project.

**Comment by doppioslash** [comment]
+1, I would back as well. I can help with setting up the crowdfunding as well.

**Comment by carlosdagos** [comment]
+1 for donation

**Comment by zpyzl** [comment]
+1 for donations, really impressive book!

**Comment by aalhour** [comment]
+1 for donations.

**Comment by Tehnix** [comment]
Just adding my voice to the choir! This is such a gem, and it is quite sad it is in this unfinished state. As far as I understood, Stephen got a job and for obvious reasons has much less time to work on this.

That said, I wonder if there is something the Haskell community could help to further the progress on this - even just a little at a time would be beneficial I think.

I don't know if Diehl himself has any thoughts on this, would be interesting to hear it :)

**Comment by lambdadog** [comment]
Hey, no commits for two years now, very interested to hear on any updates to this/possibilities for furthering it, whether by Diehl himself or by the community, would be willing to put money toward it or even contribute what little I could to it

**Comment by srid** [comment]
Meanwhile, as a momentary alternative to finishing the book, what if we put together a series of papers, tutorials, books acting as a "progression" towards understanding the theoretical basis of Haskell?

**Comment by adamwespiser** [comment]
@srid There are two good lists of foundational papers for implementing a Haskell compiler:

https://www.stephendiehl.com/posts/essential_compilers.html

https://gitlab.haskell.org/ghc/ghc/-/wikis/reading-list
